src/components/Forms/Artist/hooks/useArtistForm.ts

```diff
@@ -31,15 +32,15 @@ export default function useArtistForm(
     } = api.artist.update.useMutation()
     const deleteartistPhotoMutation = api.artist.deletePhoto.useMutation()
 
-    const isEditing = !!currentValues
+    const isEditing = editing
```

Comment:
Instead of re-assigning this var, just have the initial argument be `isEditing` please